### ISSUE TITLE
spacy_install() not working

### ISSUE THREAD

**Issue by seb-29** [quanteda/spacyr#242]
Hi, 

after installing [spacyr](https://cran.r-project.org/web/packages/spacyr/index.html) using `install.packages("spacyr")` and [miniconda3](https://docs.conda.io/en/latest/miniconda.html), I tried installing spaCy following the instructions available at https://spacyr.quanteda.io/. Thus, I ran the following commands using RStudio (with admin rights):
```
library(spacyr)
spacy_install()
```
The installation process began. However, it got stuck at the following point:
```
Solving environment: ...working... unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: ...working... unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): ...working... done
```
At this point, nothing happens anymore. Could you please help me understand what the issue is? I´m quite puzzled, as I´ve used spacyr without any problems in the past - now I seem unable to set it up!

**Comment by JBGruber** [comment]
Seems to be a duplicate of #231. You can try my solution in this https://github.com/quanteda/spacyr/issues/236#issuecomment-1702311825.

**Comment by JBGruber** [comment]
That message is emitted by `reticulate`. I'm not quite sure why it does not like the miniconda installation (it was the same for me). But I'm not a big fan of conda anyway. I don't see the advantage over the small python installation `reticulate` now offers and that one seems much more stable! So I would suggest to swap conda for python if you're not attached to it for some reason.

**Comment by JBGruber** [comment]
`spacyr` requires Python. Anaconda is Python and then some. I've never used any of the other features of Anaconda. So I think getting less functions, but more stable ones is preferable. `spaCy` is now installed in a Python environment. Which again do less then conda environments, but I had less problems with them in the past. 

But maybe someone (e.g., @kbenoit) has a different take. Then I could still integrate an alternative route through Anaconda.

**Comment by JBGruber** [comment]
The "[64-bit Git for Windows Setup](https://github.com/git-for-windows/git/releases/download/v2.42.0.windows.2/Git-2.42.0.2-64-bit.exe)" should do (although I highly suggest you look into winget or [chocolately](https://docs.chocolatey.org/en-us/choco/setup) for package management). I'm a little surprised you need this. I always assumed RStudio comes packed with a git version. But the underlying `reticulate` command might need more (?). I'll test on a virtual machine when I have time.

(BTW, I'm a political scientist :sweat_smile:)

**Comment by kbenoit** [comment]
This should be solved now with v1.3, please give it a try.
